### DISCUSSION TITLE
Are there any open-source projects similar to Huly?

### DISCUSSION THREAD

**Discussion by hnlinhorc** [hcengineering/platform#6894] [Q&A]
Hello everyone,
Are there any open-source projects similar to Huly? 
I would like to learn more about their strengths and weaknesses to see which project is the most suitable for me.

**Comment by vasani-arpit** [comment]
Depends on what is the core module you are looking for. 

I think huly is a good combination of all things required for team collaboration. 

Here is a list(not in any order) of other open source projects as per my knowledge and notes from my side

- [Openproject](https://github.com/opf/openproject): Only Tasks module is there
- [Leantime](https://github.com/Leantime/leantime): Task management and calendar planning
- [plane](https://github.com/makeplane/plane): Task management and knowledge base and storage
- [Appflowy](https://github.com/AppFlowy-IO/AppFlowy): Task management and knowledge combined like notion 
- [Ever teams](https://github.com/ever-co/ever-teams): _Not enough stars but product looks good._ Project management, time management and other integrations 
- [Tegon](https://github.com/tegonhq/tegon): something like appflowy
- [jitsi](https://github.com/jitsi/jitsi-meet): Video calls and with team
- [Worklenz](https://github.com/Worklenz/worklenz): _Fairly new repo, could be something to look out for_ Project management, time tracking, task management and templates

I personally haven't used any of them. So, can't say more about them.

**Reply by kalinga777** [reply]
Worklenz
